src/components/DuplicatesPage/DuplicatesList/DuplicatesList.js

```diff
@@ -0,0 +1,42 @@
+import DuplicatesTable from "../DuplicatesTable/DuplicatesTable";
+import * as React from "react";
+import {array, object} from "prop-types";
+import {areDuplicatesMerged} from "../../../utils/duplicates";
+import MergedItem from "../MergedItem/MergedItem";
+
+function DuplicatesList({duplicatesList, actions}) {
+
+    const getDuplicateTitle = (duplicate) => (duplicate.firstName + ' ' + duplicate.lastName);
+
+    const renderDuplicates = (duplicates) => (
+        duplicatesList.map((duplicates, index) => (
+            <div key={duplicates.id} className="duplicates-container">
+                <b className="secondary-text text-uppercase">DUPLICATE {index + 1}
+                    - {getDuplicateTitle(duplicates.data[0])}</b>
+                {areDuplicatesMerged(duplicates) ?
+                    <MergedItem duplicate={duplicates.data[0]}/> :
+                    <DuplicatesTable duplicates={duplicates} actions={actions}/>
+                }
+            </div>
+        ))
+    );
+
+    return (
+        <div className="col-md-12">
+            <h2>{duplicatesList.length ? duplicatesList.length + " Duplicates Found" : "No Duplicates Found"} </h2>
+            {renderDuplicates()}
+        </div>
+    )
+}
+
```

Comment:
duplicatesList.length ? duplicatesList.length > 0 ---> on numbers do that comparison.